EazyE2E/ElementHelper/EzTextResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EazyE2E.ElementHelper
+{
+    public class EzTextResult<T>
+    {
+        public bool NotSupported { get; set; }
+        public bool IsMixed { get; set; }
+        public T Result { get; set; }
```

Comment:
Hey @abduncan All of this code that I've done seems to work fine, but there's one behavior that I don't like but I can't seem to get around.  The class represents a result of when a text property is queried for.  There are 3 possible outcomes, Not Supported, Mixed, or an actual result.  Because this class is generic I can't have a constructor and set private variables and I don't see a way around that.  I don't like having the backing properties having getters and setters because then an end user could accidentally update the state of the results which is less than ideal.

Help me Obi-Wan Kenobi.  You're our only hope.